src/components/AudioTrack/layouts/Row.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="row-track flex flex-row" :class="`size-${size}`">
```

Comment:
From our conversation with @zackkrida , 

> each [search result] should probably also be an article tag, semantically.
?

Reply:
Should it be wrapped in an `article` or itself be an `article`?

Reply:
I think it should itself be an article, only for the sake of fewer dom nodes in the tree.

Reply:
Changed to an `article` element in c80de9e.